src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    public static final int MAX_SIDE = 100;
```

Comment:
remove this constant from this class and move it to FigureSupplier, also public -> private, MAX_SIDE -> MAX_RANDOM_NUMBER